transports/msmq/index.md

```diff
@@ -77,25 +77,24 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
 |----------------|---------------------|-----------------------------------|
 | Owning account | Send, Receive, Peek | Set by NServiceBus                |
 | Administrators | Full                | Set by NServiceBus                |
-| Anonymous      | Send                | Set by NServiceBus prior to 6.1.0 |
-| Everyone       | Send                | Set by NServiceBus prior to 6.1.0 |
+| Anonymous      | Send                | Set by NServiceBus in Versions 6.0.x and below |
+| Everyone       | Send                | Set by NServiceBus in Versions 6.0.x and below |
 
+NOTE: Starting with Versions 6.1.0 and above, the installers will not automatically grant permissions to the `Anonymous` and `Everyone` group. It will respect the existing queue permissions that have been set up for the endpoint. 
 
-When an endpoint sends a command or event message that source requires permission at the target queue. For endpoints using the either the publish-subscribe or request-response patterns, both parties actually send messages and must have permission to send messages to each other. This is to allow the Subscribe message to be sent from the subscribing endpoint to the publishing endpoint in case of publish-subscribe or the response to be send back to the client in case of request-response.
+Any endpoint that sends a message to a target endpoint requires the `Send` permission to be granted for the sending user account on the target queue. For example, if an `endpoint A` is running as `userA` and is sending a message to `endpoint B`, then `userA` requires the `Send` permission to be granted on `endpoint B`'s queue. When using messaging patterns like request-response or publish-subscribe, the queues for both the sending endpoint and the target endpoint will require `Send` permissions to be granted to each others user accounts. 
```

Comment:
> the queues for both the sending endpoint and the target endpoint will require

It's not clear which is the "sending" and "target" endpoints in a pub-sub/req-res interaction. I would just say "both"